evaluator/builtin_time.go

```diff
@@ -390,6 +390,16 @@ func builtinCurrentTime(args []types.Datum, _ context.Context) (d types.Datum, e
 	return convertToDuration(d, fsp)
 }
 
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_utc-date
+func builtinUTCDate(args []types.Datum, _ context.Context) (d types.Datum, err error) {
+	year, month, day := time.Now().Date()
+	t := mysql.Time{
+		Time: time.Date(year, month, day, 0, 0, 0, 0, time.Local),
```

Comment:
@shenli It does not work.

time.UTC:
```
mysql> select utc_date+1;
+----------------+
| utc_date+1     |
+----------------+
| 20160401000001 |
+----------------+
1 row in set (0.00 sec)
```

And please take a look **current_date**:

**TiDB**:
```
mysql> select utc_date+1, current_date+1;
+----------------+----------------+
| utc_date+1     | current_date+1 |
+----------------+----------------+
| 20160401000001 | 20160401000001 |
+----------------+----------------+
1 row in set (0.00 sec)
```

**MySQL**:

```
mysql> select utc_date+1, current_date+1;
+------------+----------------+
| utc_date+1 | current_date+1 |
+------------+----------------+
|   20160402 |       20160402 |
+------------+----------------+
1 row in set (0.00 sec)
```